analyses/molecular-subtyping-HGG/05-HGG-molecular-subtyping-fusion.Rmd

```diff
@@ -72,11 +72,16 @@ Thus, we will look at _any_ instances of fusions that contain _FGFR1_ or _NTRK_.
 
 ```{r}
 FGFR1_fusions <- fusion_df %>%
-  filter(str_detect(FusionName, "FGFR1"))
+  filter(str_detect(FusionName, "FGFR1")) %>% 
+  select(Sample, FusionName) %>% 
+  group_by(Sample) %>% 
+  summarize(FGFR1_fusions = paste(sort(unique(FusionName)), collapse = ", ")) 
 
 dim(FGFR1_fusions)
 ```
 
+more than one _FGFR1_fusion presents, code is changed accordingly
```

Comment:
It is OK to amend the old text - can just remove this old text.

Can you also add after line 78 and 92: (there are dup rows for NTRK, so may as well take care of both here)
``` 
%>%
unique()
```